src/components/Pagination/Pagination.scss

```diff
@@ -29,3 +28,6 @@
     @extend .disabled;
   }
 }
+.page-item.disabled .page-link {
+  background-color: transparent;
+}
```

Comment:
Can you share the context on why this CSS change was needed?

Reply:
Yes; it's to override some CSS for making all the list items look similar. The disabled list items in this context are the ellipses representing ranges of pages not visible. The design of the paragon component is that those elements shouldn't have a distinct background color, and should blend in with the surrounding background color.

With that said I am _quite_ bad at CSS/SCSS; is this the use-case for a SCSS variable override? It def looks like I could've put it above in the `.disabled { ... .page-link { <here> } }` above and achieved the same degree of specificity. I'll try that out for my next push.